portfolio/src/main/webapp/script.js

```diff
@@ -14,21 +14,60 @@
 
 /**
  * Adds a random fact to the page. 
- * TODO: Currently not in use on portfolio will utilize it later.
  */
 function addRandomFact() {
-  const facts =
-      [
-          'I am learning to play the piano!', 
-          'I am from the Commonwealth of Dominica.',
-          'I love Gouda Cheese.',
-          'I love reading web novels.'
-        ];
-
-  // Pick a random fact.
-  const fact = facts[Math.floor(Math.random() * facts.length)];
-
-  // Add it to the page.
-  const factContainer = document.getElementById('fact-container');
-  factContainer.innerText = fact;
-}
+    const facts =
+        [
+            'I am learning to play the piano!', 
+            'I am from the Commonwealth of Dominica.',
+            'I love Gouda Cheese.',
+            'I love reading web novels.',
+          ];
+  
+    // Pick a random fact.
+    const fact = facts[Math.floor(Math.random() * facts.length)];
+  
+    // Add it to the page.
+    const factContainer = document.getElementById('fact-container');
+    factContainer.innerText = fact;
+  }
+
+
+/** 
```

Comment:
For better management, You can split the slide show functions into a different js file. Since the project scope is less, I guess it's fine to have it included in this file.

Reply:
Alright, great. Added screenshot links in this PR and in a later PR will separate the slideshow functions in its own file, thanks.